app/controllers/customers_controller.rb

```diff
@@ -0,0 +1,12 @@
+class CustomersController < ApplicationController
+  def show
+    @customer = Customer.find(params[:id])
+  end
+
+  def create
+    item_id = params[:item_id].to_i
+    item = Item.find(item_id)
+    @customer_item = CustomerItem.create(customer: @customer, item: item)
```

Comment:
In Rails, @instance variables are used a bit differently in controllers/views than in other types of files. The @customer that you made above on line 3 is going to be available in the view for the show.index.html but ONLY for that view. The @customer instance variable isn't accessible in the `create` action so you have to create it again by finding the Customer record for the given customer ID like you did with item. 

I'd also suggest moving this `create` action to a different controller to indicate that it's creating a *joins* record. You could have this create in a CustomerItemsController to indicate you're building the relationship between a given customer and a given item. This is more RESTful than using the create in the CustomersController because we want to reserve that `create` for when you're creating a new customer (not a joins record)